src/backend/routes/api/transport.ts

```diff
@@ -77,10 +77,17 @@ router.post('/transport/image', imageUploader, async (req: Request, res: Respons
     return;
   }
 
+  const fileData = {
+    ...req.files.image[0],
+    url: '/uploads/' + req.files.image[0].filename,
```

Comment:
why do we need to hardcode the URL at the Database instead of using the file id of the name?

Reply:
because all static files are serving from uploads directory
before this change file url depends on uploads folder name, which is wrong

Reply:
but why do we need to store the `/uploads/` at the db? What if we will change it in the future? I think we can compose the URL on the fly by the client-side app.

Reply:
I guess you're right, we can resolve it in the separate issue
With this change I just resolve issue when uploads stores in folder with different name